modules/apps/portal-vulcan/portal-vulcan-test/src/testIntegration/java/com/liferay/portal/vulcan/internal/jaxrs/context/provider/test/SortContextProviderTest.java

```diff
@@ -122,27 +119,6 @@ public void testCreateContextThrowsInvalidSortException() throws Exception {
 				_mockResource));
 	}
 
-	@Test(expected = NotAcceptableException.class)
```

Comment:
In this case, as with the GET method, I think we should rewrite the test (I didn't know when I started reviewing the PR that for the methods != GET, an exception would not be thrown). Is it possible to rewrite the test to verify for the GET method that an exception is NOT thrown and for a different method that the exception is still being thrown?